CellProfiler_pipelines/Pipelines/NF1_analysis_cp_plugin.cppipe

```diff
@@ -2,7 +2,7 @@ CellProfiler Pipeline: http://www.cellprofiler.org
 Version:5
```

Comment:
How is this file used in conjunction with `NF1_analysis.cpipe`? They seem to have nearly identical changes in this PR.

Reply:
Since in this repo we have multiple different image-analysis pipeline, we have to have two different analysis cppipe files to run different segmentation method. Essentially, the two analysis pipelines need to be the extract same except for segmentation to be able to perform benchmarking in the future.